src/main/java/org/jabref/gui/openoffice/StyleSelectDialog.java

```diff
@@ -272,9 +271,11 @@ private void setupPopupMenu() {
 
         // Create action listener for removing a style, also used for the remove button
         removeAction = actionEvent -> getSelectedStyle().ifPresent(style -> {
-            if (!style.isFromResource() && (JOptionPane.showConfirmDialog(diag,
-                    Localization.lang("Are you sure you want to remove the style?"), Localization.lang("Remove style"),
-                    JOptionPane.YES_NO_OPTION) == JOptionPane.YES_OPTION)) {
+            boolean removeStylePressed = frame.getDialogService().showConfirmationDialogAndWait(Localization.lang("Remove style"),
+                    Localization.lang("Are you sure you want to remove the style?"),
+                    Localization.lang("Remove style"),
+                    Localization.lang("Cancel"));
+            if (!style.isFromResource() && removeStylePressed) {
```

Comment:
previously the dialog was only shown if `style.isFromResource` returned false. Is this change of behavior desirable?

Reply:
Ah shit, I oversaw that condiation